services/apps/src/apps/clients/iplant_groups.clj

```diff
@@ -18,6 +18,30 @@
 (def grouper-analysis-permission-def-fmt "iplant:de:%s:analyses:analysis-permission-def")
 (def grouper-analysis-resource-name-fmt "iplant:de:%s:analyses:%s")
 
+(def ^:private permission-precedence
+  (into {} (map-indexed (fn [i v] (vector v i)) ["own" "write" "read"])))
```

Comment:
I like this solution, especially if you needed to build this map dynamically with values read from a db, but since they're static in the code, would it be more readable if the final map was just static?
```clojure
{"own"   0
 "write" 1
 "read"  2}
```